### ISSUE TITLE
Tracerwaring问题---轉為TVM

### ISSUE THREAD

**Issue by andrewwang0612** [murdockhou/Yet-Another-EfficientDet-Pytorch-Convert-ONNX-TVM#15]
不好意思，想要請教您
我在轉換為TVM時，出現了以下的warning，TVM的版本為0.9
所以在您下面需要修改的部分已經有些不一樣了
/home/youwh/ywh/efficientDet--ONNX/efficientnet/utils_extra.py:37: TracerWarning: Converting a tensor to a NumPy array might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
h, w = x.cpu().detach().numpy().shape[-2:]
我不理解它是否會有影響，想詢問一下您這個警告是否有可能會對接下的inference有影響 ?
非常感谢您能回覆！
